circle/controllers/user_contorllers.go

```
@@ -41,27 +43,21 @@ func Getemail(ee string,VerificationCode string)  {
 	smtpHost := "smtp.qq.com"                                           
 	smtpPort := "587"                                                             
 	auth := smtp.PlainAuth("", "2388287244@qq.com", m, smtpHost)
-	e.Send(smtpHost+":"+smtpPort, auth) 
+	e.Send(smtpHost+":"+smtpPort, auth)  
```

golangci-lint (1.62.2)

Error return value of `e.Send` is not checked

(errcheck)